src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf");
+}
```

Comment:
You should add separate @font-face for each font style (one for normal, one for bold)